Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM node:8
+FROM node:16
```

Comment:
Can we jump to node 18 here as its the most recent security patched version? https://endoflife.date/nodejs

Reply:
did you one better - node 20.